.github/ISSUE_TEMPLATE/enhancement.md

```diff
@@ -0,0 +1,20 @@
+---
+name: "\U00002728Enhancement"
+labels: "Type: Enhancement"
+about: Something isn't broken, but should be made better when we have some time.
+
+---
+
+# Enhancement
+
+## What needs done
```

Comment:
Optional: Consider breaking this down into something like "Success Criteria" and "Proposed Steps".

I think "Success Criteria" is a helpful framing when you're working on someone else's request, and might be more stable than the steps suggested when the issue was created.